### PR TITLE
Fix issue with tts config for PreOreoAttributes

### PR DESCRIPTION
**Pull request by VitaminPSG**
Engine.KEY_PARAM_STREAM value should be Int or Long which

```
<changelog>Fix issue: Synthesis request parameter streamType contains value  with invalid type</changelog>
```

### CODE REVIEW

File: libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsPlayerAttributes.kt [hunk 1]
```diff
@@ -61,9 +61,9 @@ sealed class VoiceInstructionsPlayerAttributes {
 
         override fun configureTextToSpeech(): TextToSpeech.(Bundle) -> Unit {
             return { bundle ->
-                bundle.putString(
+                bundle.putInt(
```
Comment (Guardiola31337) [inline libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsPlayerAttributes.kt:64]: Can we add a unit test covering this logic?

Comment (VitaminPSG) [reply libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsPlayerAttributes.kt:64]: Yes, please have a look

**Comment by Guardiola31337** [comment]
@VitaminPSG could you please squash the commit history when you have a chance? 🙏

**Review by Guardiola31337 — APPROVED** [review]
Thanks for addressing the feedback @VitaminPSG :shipit:

**Pull request merged**